automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/Leonardo.scala

```diff
@@ -177,6 +177,12 @@ object Leonardo extends RestClient with LazyLogging {
       val cookie = Cookie(HttpCookiePair("LeoToken", token.value))
       handleContentItemResponse(parseResponse(getRequest(url + path, httpHeaders = List(cookie))))
     }
+
+    def setCookie(googleProject: GoogleProject, clusterName: ClusterName)(implicit token: AuthToken, webDriver: WebDriver): String = {
+      val path = notebooksPath(googleProject, clusterName) + "/setCookie"
+      logger.info(s"Set cookie: GET /$path")
+      parseResponse(getRequest(url + path))
```

Comment:
So this sends a `/setCookie` request _without_ a token, which should always return 403. I'm wondering if this method should actually pass the token that it receives. The test below could explicitly pass it a bad token. It makes this method more generic and reusable.

I think this should work:
```
parseResponse(getRequest(url + path, httpHeaders = List(Authorization(OAuth2BearerToken(token.value)))))
```